src/modules/services/components/ClientServicesPage.tsx

```diff
@@ -36,41 +40,41 @@ const ClientServicesPage: React.FC<{
     () =>
       (
         [
-          SERVICE_BASIC_COLUMNS.dateProvided,
+          SERVICE_BASIC_COLUMNS.serviceDate,
           SERVICE_BASIC_COLUMNS.serviceType,
           {
             key: 'project',
             header: 'Project Name',
-            render: (row) => (
-              <RouterLink
-                to={[
-                  generateSafePath(
-                    EnrollmentDashboardRoutes.ENROLLMENT_OVERVIEW,
-                    {
-                      enrollmentId: row.enrollment.id,
-                      clientId,
-                    }
-                  ),
-                  'services',
-                ].join('#')}
-              >
-                {row.enrollment.projectName}
-              </RouterLink>
-            ),
-          },
-          {
-            key: 'en-period',
-            header: 'Enrollment Period',
-            optional: true,
-            render: (row) => (
-              <EnrollmentDateRangeWithStatus enrollment={row.enrollment} />
-            ),
+            render: (row) => row.enrollment.projectName,
           },
           {
             ...SERVICE_COLUMNS.serviceDetails,
             optional: true,
             defaultHidden: true,
           },
+          {
+            ...BASE_ACTION_COLUMN_DEF,
+            render: (row) => (
+              <TableRowActions
+                record={row}
+                recordName={`${getServiceTypeForDisplay(row.serviceType)} on ${parseAndFormatDate(row.dateProvided)}`}
+                primaryActionConfig={getViewServiceMenuItem(
+                  row,
+                  row.enrollment.id,
+                  clientId
+                )}
+                secondaryActionConfigs={[
+                  {
+                    ...getViewEnrollmentMenuItem(row.enrollment, {
+                      id: clientId,
```

Comment:
This is the use case for https://github.com/greenriver/hmis-frontend/pull/989#discussion_r1901300319